### PR TITLE
Migration docs

### PR DESCRIPTION
**Pull request by pferreir**

### CODE REVIEW

File: docs/source/installation/upgrade.rst [hunk 1]
```diff
@@ -1,4 +1,203 @@
...
+    generated automatically).
+
+
+Post-migration work
++++++++++++++++++++
+
+After the migration is done you may need to apply some adjustments in your ``indico.conf``. You may want to read our
+guide on how to configure an Identity/Authentication provider.
+
+We really recommend as well that you move your old Indico archive (``/opt/indico-legacy/achive``) inside your new
```
Comment (ThiefMaster) [inline docs/source/installation/upgrade.rst]: typo

File: docs/source/installation/upgrade.rst [hunk 2]
```diff
@@ -1,4 +1,203 @@
...
+
+.. code-block:: shell
+
+    zdaemon -C /opt/indico/etc/zdctl.conf status
+
+
+Moving legacy data
+++++++++++++++++++
+
+Indico 2.0 will use a directory structure that is similar to Indico 1.x, so you will need first of all rename the old
```
Comment (nop33) [inline docs/source/installation/upgrade.rst]: > so you will need first of all rename the old

**so first of all you will need to** rename the old...

File: docs/source/installation/upgrade.rst [hunk 3]
```diff
@@ -1,4 +1,203 @@
...
+
+.. code-block:: shell
+
+   pip install indico-migrate
+
+
+``indico-migrate`` requires a series of parameters that have to be tuned according to your current setup. We now provide
+a list of values that should work in most standard Indico installations. However, please **carefully read** the
+`documentation of the indico-migrate command <https://github.com/indico/indico-migrate#migration-settings>`_, to make
+sure no option conflicts with your setup.
```
Comment (nop33) [inline docs/source/installation/upgrade.rst]: > to make sure no option conflicts with your setup

to make sure **there are** no option conflicts with your setup

Comment (ThiefMaster) [inline docs/source/installation/upgrade.rst]: there's also a verb "to conflict" ;)

**Pull request closed without merging**